core/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/visualization/treemap/TreeMapOpDesc.scala

```diff
@@ -0,0 +1,103 @@
+package edu.uci.ics.amber.operator.visualization.treemap
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kjetland.jackson.jsonSchema.annotations.JsonSchemaTitle
+import edu.uci.ics.amber.operator.PythonOperatorDescriptor
+import edu.uci.ics.amber.core.tuple.{AttributeType, Schema}
+import edu.uci.ics.amber.core.workflow.OutputPort.OutputMode
+import edu.uci.ics.amber.core.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.amber.operator.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.amber.operator.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.amber.operator.visualization.hierarchychart.HierarchySection
+
+class TreeMapOpDesc extends PythonOperatorDescriptor {
+
+  @JsonProperty(required = true)
+  @JsonSchemaTitle("Attribute")
```

Comment:
- "Attribute" is too generic a name and may confuse the user, especially when you also have "Attribute Name" in the hierarchy list. Please come up with more intuitive names for both properties so it can be immediately clear to a user what purpose each property serves.
- Please add some descriptions to all the properties, like most other visualization operators do. Without a description on the frontend, it is hard to use these operators.